examples/quickstart/quickstart.ipynb

```diff
@@ -196,7 +197,7 @@
     "id": "ma53mucU0yF3"
    },
    "source": [
-    "We then add a `Trainer` step, that takes the normalized data and trains a Keras classifier on the data. Note that the `Output[ModelArtifact]` type helps in writing the model out to our artifact store. "
+    "We then add a `Trainer` step, that takes the normalized data and trains a Keras classifier on the data. Note that the model is not explicitely saved within the step. Under the hood ZenML uses Materializers to automatically persist the Artifacts that result from each step into the Artifact Store."
```

Comment:
```suggestion
    "We then add a `Trainer` step, that takes the normalized data and trains a Keras classifier on the data. Note that the model is not explicitly saved within the step. Under the hood ZenML uses Materializers to automatically persist the Artifacts that result from each step into the Artifact Store."
```

Reply:
Should we then link to materializers in the docs core concepts here?